docs-ref-services/preview/resourcemanager-streamanalytics-readme.md

```diff
@@ -107,7 +106,7 @@ For details on contributing to this repository, see the [contributing guide](htt
 <!-- LINKS -->
 [survey]: https://microsoft.qualtrics.com/jfe/form/SV_ehN0lIk2FKEBkwd?Q_CHL=DOCS
 [docs]: https://azure.github.io/azure-sdk-for-java/
-[jdk]: https://docs.microsoft.com/java/azure/jdk/
+[jdk]: /java/azure/jdk/
```

Comment:
Should this be the absolute link? https://learn.microsoft.com/en-us/azure/developer/java/fundamentals/java-support-on-azure